core/include/mm/core_memprot.h

```diff
@@ -67,17 +67,23 @@ bool core_pbuf_is(uint32_t flags, paddr_t pbuf, size_t len);
 bool core_vbuf_is(uint32_t flags, const void *vbuf, size_t len);
 
 /*
- * Translate physical address to virtual address using specified mapping
+ * Translate physical address to virtual address using specified mapping.
+ * Also tries to find proper mapping which have counterpart translation
+ * for specified length of data starting from given physical address.
+ * If length doesn`t matter or cannot be precisely defined, then len
```

Comment:
`cannot be precisely defined` and `hard to specify` (below) sound strange.
Should one set the `len` of not? 
Maybe rephrase to something like: 'Can be 1 if caller knows the area does not spread over mapping granule'.

Reply:
You`re right, rephrased this.